package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "less-variable-resolver",
-  "version": "0.1.0",
+  "version": "0.1.2",
```

Comment:
`0.1.3` unless you want to create an extra commit for that.

Reply:
Or maybe... we don't publish b/c of this (lock-only) change and the mere syncing of package file and lock file is just fine.